integration/netmon/netmon_test.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Environment variables
+IMAGE="busybox"
+NETWORK_NAME="test"
+CONTAINER_NAME="containerA"
+PAYLOAD_ARGS="tail -f /dev/null"
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+	extract_kata_env
+
+	# Enabling netmon at the configuration file
+	sed -i 's|#enable_netmon = true|enable_netmon = true|' ${RUNTIME_CONFIG_PATH}
+}
+
+@test "test netmon" {
+	docker network create $NETWORK_NAME
+	docker run --runtime=$RUNTIME -d --name $CONTAINER_NAME $IMAGE sh -c $PAYLOAD_ARGS
+	docker network connect $NETWORK_NAME $CONTAINER_NAME
+	# Get ip address
+	run docker inspect --format "{{.NetworkSettings.IPAddress}}" $CONTAINER_NAME
+	[ "$status" -eq 0 ]
+}
+
+teardown() {
+	clean_env
+
+	extract_kata_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+	docker network disconnect $NETWORK_NAME $CONTAINER_NAME
```

Comment:
You probably want to do this before calling `clean_env` above, as I suspect that deleted your container...

Reply:
@grahamwhaley done :)